signer/rsapss/rsapss.go

```diff
@@ -64,6 +66,20 @@ func New(conf signer.Configuration) (s *RSAPSSSigner, err error) {
 	if !ok {
 		return nil, errors.Errorf("rsapss: unsupported public key type %T, use RSA keys", s.pubKey)
 	}
+	if conf.RSAPSSSaltLengthEqualsHash {
+		s.rsaPssSaltLength = rsa.PSSSaltLengthEqualsHash
+	} else {
+		s.rsaPssSaltLength = rsa.PSSSaltLengthAuto
+	}
+	switch conf.RSAPSSHash {
+	case "sha1":
+		s.rsaPssHash = crypto.SHA1
+	case "sha256":
+		s.rsaPssHash = crypto.SHA256
+	default:
+		// default to sha256
+		s.rsaPssHash = crypto.SHA256
```

Comment:
nit: existing signers use SHA1 so I'd prefer this to error as misconfigured instead or default to SHA1